### PR TITLE
AB#6734: Home Page hero from CH content

### PR DESCRIPTION
**Pull request by scottmulligan**
- Updated home page hero styles to match new design
- Home page hero now pulling content from CH datasource item
- CTA button in hero is moved to a separate component with datasource in placeholder on home component

### CODE REVIEW

File: Website/src/items/content/EdgeWebsite/home.yml
Comment (jflheureux) [inline Website/src/items/content/EdgeWebsite/home.yml:16]: Please review the renderings for this page. It is not normal for the page to delete everything from its template standard values to re-add everything.

File: Website/src/items/templates/edgewebsite/HeroSectionCta/__Standard Values.yml
Comment (jflheureux) [inline Website/src/items/templates/edgewebsite/HeroSectionCta/__Standard Values.yml:1]: Does this template really need standard values?

File: Website/src/rendering/src/components/HeroSection.tsx
Comment (jflheureux) [inline Website/src/rendering/src/components/HeroSection.tsx:5]: What is the reason for this change of base props? I do not see `props.children` used anywhere in this component.

File: Website/src/rendering/src/components/HeroSection.tsx
Comment (jflheureux) [inline Website/src/rendering/src/components/HeroSection.tsx:3]: This should be merged with the first import line from the same NPM package.

File: Website/src/rendering/src/components/HeroSection.tsx
Comment (jflheureux) [inline Website/src/rendering/src/components/HeroSection.tsx:37]: The name of this placeholder is not following the other placeholders naming convention (`jss-[component-name]-content`. For this component, it should be `jss-hero-section-content`. Please also change the placeholder name in the content tree.

File: Website/src/rendering/src/components/HeroSectionCta.tsx
Comment (jflheureux) [inline Website/src/rendering/src/components/HeroSectionCta.tsx:4]: This type name should match the component: `HeroSectionCtaProps`

File: Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml
Comment (jflheureux) [inline Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml:4]: The "Hero" component datasource template does not seem to be used anymore but it was not deleted. Could you please check that is the case and delete it if needed? Thanks.

File: Website/src/rendering/src/components/HeroSection.tsx
Comment (scottmulligan) [reply Website/src/rendering/src/components/HeroSection.tsx]: Updated

File: Website/src/rendering/src/components/HeroSection.tsx
Comment (scottmulligan) [reply Website/src/rendering/src/components/HeroSection.tsx]: Updated

File: Website/src/rendering/src/components/HeroSectionCta.tsx
Comment (scottmulligan) [reply Website/src/rendering/src/components/HeroSectionCta.tsx]: Updated

File: Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml
Comment (scottmulligan) [reply Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml:4]: Good one - I have removed the original template that was used as datasource items for the hero component.

File: Website/src/items/content/EdgeWebsite/home.yml
Comment (scottmulligan) [reply Website/src/items/content/EdgeWebsite/home.yml]: Success! Manually updated "__Renderings".

File: Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml
Comment (jflheureux) [reply Website/src/items/content/EdgeWebsite/home/Page Components/Hero.yml:4]: While that should work and is OK for now, this is not how this page rendering XML value should look like. It should not redefine all placeholders. It should not mention the header, MainNavigation, Footer, SponsorsGrid (and its section) and NewsGrid (and its section) components.

**Comment by jflheureux** [comment]
Since templates were changed, you must run `pre-pr-updates.ps1` to regenerate the GraphQL introspection data. This must be done with a running local Sitecore instance.

File: Website/src/items/templates/edgewebsite/HeroSectionCta.yml
Comment (jflheureux) [inline Website/src/items/templates/edgewebsite/HeroSectionCta.yml:5]: Since templates were changed, you must run `pre-pr-updates.ps1` to regenerate the GraphQL introspection data. This must be done with a running local Sitecore instance.

Comment (scottmulligan) [reply Website/src/items/templates/edgewebsite/HeroSectionCta.yml:5]: 100% forgot about this. Thanks!

File: Website/src/rendering/src/components/HeroSection.tsx
Comment (jflheureux) [inline Website/src/rendering/src/components/HeroSection.tsx:18]: I am not able to test the PR on a portal instance as the image does not load because of the CH port issue. Should we wait for this issue to be resolved before we merge this PR?

Comment (jflheureux) [reply Website/src/rendering/src/components/HeroSection.tsx:18]: I was able to add the port by editing the field raw value. It should work once the port issue is fixed.

**Review by jflheureux — APPROVED** [review]

**Pull request merged**
